docs/examples/android.md

```diff
@@ -1,8 +1,8 @@
 Android Examples
 ================
 
-This page will give a few example to get you up and running with
-browsertime on Android devices.  Generally, add `--android` and then
+This page will give a few examples to get you up and running with
+Browsertime on Android devices.  Generally, add `--android` and then
```

Comment:
Hmm, I haven't been capitalizing browsertime.  I was writing it as `browsertime`, then dropped that.  Do you capitalize Geckodriver?  I haven't been.  I don't capitalize `ls` (even when I write it ls).

I don't feel strongly about this.

Reply:
Thanks for the review, and for the question, @ncalexan!

I would and do capitalize Browsertime (per https://github.com/sitespeedio/browsertime/blob/d7cbe43439cbb4d7b0b44f028387d4a1697dbb0f/README.md and elsewhere), but not geckodriver (per https://github.com/mozilla/geckodriver/blob/79bc0ed40fae4454749530fef9f2a6c5c0e50b59/README.md).

I would and do also capitalize WebDriver (not that you asked, but as an additional example).

I'm fine leaving well enough alone on the s/browsertime/Browsertime changes, if you'd like me to drop those?

Reply:
(And I should say I strive to leave binary or command-line, etc., names alone, capitalization-wise, for hopefully obvious reasons.)

Reply:
Fixed; went with prevailing style.